crates/client/src/https_client_connection.rs

```diff
@@ -37,16 +38,19 @@ impl<T> HttpsClientConnection<T> {
     /// # Arguments
     ///
     /// * `name_server` - IP and Port for the remote DNS resolver
+    /// * `bind_addr` - IP and port to connect from
     /// * `dns_name` - The DNS name, Subject Public Key Info (SPKI) name, as associated to a certificate
     /// * `client_config` - The TLS config
     #[allow(clippy::new_ret_no_self)]
     pub fn new(
         name_server: SocketAddr,
+        bind_addr: Option<SocketAddr>,
```

Comment:
Given the public interface, should we avoid the breaking API change with a new method instead? `new_with_bind_addr`?

Reply:
Done